airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/OpenTelemetryMetricClient.java

```diff
@@ -39,6 +44,9 @@ public class OpenTelemetryMetricClient implements MetricClient {
   private Meter meter;
   private SdkMeterProvider meterProvider;
 
+  private final Map<String, ObservableDoubleGauge> gauges = new HashMap<>();
+  private final Map<String,Map<Attributes,Double>> gaugeValues = Collections.synchronizedMap(new HashMap<>());
```

Comment:
nit: insert space in between 

`Map<String, Map<Attributes, Double>>`

Reply:
reformatted both files according to checkstyle.xml